install/kubernetes/helm/istio/charts/grafana/templates/grafana-ports-mtls.yaml

```diff
@@ -1,4 +1,3 @@
-{{ define "grafana-default.yaml.tpl" }}
```

Comment:
Why removing this?

Reply:
We can directly create the resource, no need to package it in a template.